itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/TestMetaStoreLimitPartitionRequest.java

```diff
@@ -73,6 +73,9 @@ public static void beforeTest() throws Exception {
     conf.setBoolVar(HiveConf.ConfVars.METASTORE_TRY_DIRECT_SQL, true);
     conf.setBoolVar(HiveConf.ConfVars.DYNAMIC_PARTITIONING, true);
     conf.setBoolVar(HiveConf.ConfVars.HIVE_CBO_ENABLED, false);
+    // Disable loading dynamic partitions from partition names in this test
+    // because get_partitions_by_names will also hit partition limit since HIVE-28062.
+    conf.setBoolVar(HiveConf.ConfVars.HIVE_LOAD_DYNAMIC_PARTITIONS_SCAN_SPECIFIC_PARTITIONS, false);
```

Comment:
`hive.load.dynamic.partitions.scan.specific.partitions` was inroduced by HIVE-25178 to fetch only specific relevant **external** tables partitions by partition names(using `getPartitionsByNames `method):
https://github.com/apache/hive/blob/ffb1165f59defa66b31b4fd9cb6367b71050071b/ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java#L3187-L3202

Here setting `hive.load.dynamic.partitions.scan.specific.partitions` to false will give others a false impression that this propertity can be used to control the partitions limit. However, It's just a coincidence as it just skip to  invoke `getPartitionsByNames `method when this propertity is false, and it has nothing to do with partitions limit.

Am i right?
If it is ok, i would suggest to think another better to fix this.

Reply:
It's an option to determine whether use `getPartitionsByNames` to load partitions, in this test we disable it because  the setup of this test would be failed by partition limit, which is not related to the real tests actually.

Reply:
I know it's just a workaround to fix the failed UT and not related to the real cases. But IMO, `hive.load.dynamic.partitions.scan.specific.partitions` should not be bound together with `getPartitionsByNames `, and we may can use other methods like `getPartitionsBy...` to achieve this in future.
This part maks me little confusing, but this is just a small fix about the UT, so i think it does't matter. :)